VAMobile/src/components/VAIcon/VAIcon.test.tsx

```diff
@@ -1,11 +1,5 @@
-import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
-
-import { context, render, RenderAPI, waitFor } from 'testUtils'
-import HomeSelected from './navIcon/homeSelected.svg'
+import { context, render, screen } from 'testUtils'
 import VAIcon from './VAIcon'
 import { InitialState } from 'store/slices'
```

Comment:
fairly certain initialState can be removed like your part 4 ticket

Reply:
Updated